internal/outputs/http.go

```diff
@@ -44,14 +48,16 @@ func postRequest(url string, key string, data []byte) error {
 	resp, err := client.Do(req)
 	if err != nil {
 		load.Logrus.WithError(err).Error("http: failed to send")
+		return errors.Wrap(err, "http: failed to send")
 	}
-	if resp == nil {
-		return fmt.Errorf("http: response nil")
-	}
-	defer resp.Body.Close()
+
+	defer func(body io.ReadCloser) {
+		_, _ = ioutil.ReadAll(body)
```

Comment:
do we need to read the whole body before closing?

Reply:
It is good practice to read the all the body of a http request made in Go.
>Response will contain a non-nil Body which the user is expected to close. If the Body is not both read to EOF and closed, the Client's underlying RoundTripper (typically Transport) may not be able to re-use a persistent TCP connection to the server for a subsequent "keep-alive" request.
https://golang.org/pkg/net/http/#Client.Do

Just making sure we tidy up resources we've used.